src/emulator/gui/src/settings_dialog.cpp

```diff
@@ -30,7 +30,43 @@ void draw_settings_dialog(HostState &host) {
     ImGuiTabBarFlags settings_tab_flags = ImGuiTabBarFlags_None;
     ImGui::BeginTabBar("SettingsTabBar", settings_tab_flags);
 
+    // Core
+    if (ImGui::BeginTabItem("Core")) {
+        ImGui::PopStyleColor();
+        ImGui::Text("LLE Module \nSet your LLE Module Wanted.");
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Write exemple 'libpgf', press Apply and re write other.");
```

Comment:
ImGui::SetTooltip("For example, libpgf is a LLE module. To activate any module, select it and press Apply.");

Reply:
Wrong, is not select for write, need write manually lle name

Reply:
So `ImGui::SetTooltip("For example, libpgf is a LLE module. To activate any module, write it and press Apply.");`?

Reply:
Hmm yes ^^